rdr_service/alembic/versions/00dd4c332eae_adding_w2sc_attributes.py

```diff
@@ -0,0 +1,50 @@
+"""adding_w2sc_attributes
+
+Revision ID: 00dd4c332eae
+Revises: 7269de0a691a
+Create Date: 2022-03-02 10:21:33.868462
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '00dd4c332eae'
+down_revision = '7269de0a691a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('genomic_set_member', sa.Column('cvl_w2sc_manifest_job_run_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'genomic_set_member', 'genomic_job_run', ['cvl_w2sc_manifest_job_run_id'], ['id'])
+    # ### end Alembic commands ###
```

Comment:
We are going to want to create a raw table for every manifest we ingest going forward. This one can be called: `genomic_w2sc_raw`. Maybe that should be a separate PR though.

Reply:
Yea will make a separate PR for this. What about the manifest generations as well?

Reply:
Yea, it's probably better to have and not need than need and not have.